### PR TITLE
changelog for 4.0 beta

### PR DESCRIPTION
**Pull request by minrk**
stepping toward #4350

### CODE REVIEW

File: docs/source/reference/changelog.md [hunk 1]
```diff
@@ -8,6 +8,109 @@ command line for details.
...
+- fix bracket typo in capacity figures [#4250](https://github.com/jupyterhub/jupyterhub/pull/4250) ([@minrk](https://github.com/minrk), [@consideRatio](https://github.com/consideRatio))
+- convert remaining rst files to myst [#4249](https://github.com/jupyterhub/jupyterhub/pull/4249) ([@minrk](https://github.com/minrk), [@consideRatio](https://github.com/consideRatio))
+- doc: fix formatting of spawner env-vars [#4245](https://github.com/jupyterhub/jupyterhub/pull/4245) ([@manics](https://github.com/manics), [@consideRatio](https://github.com/consideRatio), [@minrk](https://github.com/minrk))
+
+#### API and Breaking Changes
+
+- require sqlalchemy 1.4 [#4319](https://github.com/jupyterhub/jupyterhub/pull/4319) ([@minrk](https://github.com/minrk), [@manics](https://github.com/manics))
+- Use XSRF tokens for cross-site checks [#4032](https://github.com/jupyterhub/jupyterhub/pull/4032) ([@minrk](https://github.com/minrk), [@consideRatio](https://github.com/consideRatio), [@julietKiloRomeo](https://github.com/julietKiloRomeo))
```
Comment (consideRatio) [inline docs/source/reference/changelog.md:103]: What do you think about relocating these to come beore new / bugs / maint / docs ?

File: docs/source/reference/changelog.md [hunk 2]
```diff
@@ -8,6 +8,109 @@ command line for details.
 
 ## [Unreleased]
 
+### 4.0 (beta) - 2023-XX-YY
```
Comment (consideRatio) [inline docs/source/reference/changelog.md:11]:
```suggestion
### 4.0 (beta) - 2023-XX-YY
```

If we are to merge this now, the date ought to be updated.

**Review by consideRatio — APPROVED** [review]
The text summarizing changes was great! Marking this as approved but leaving the merge to you to be accompanied with cutting the beta release if you think its time.

File: docs/source/reference/changelog.md (hunk 1 above)
Comment (minrk) [reply docs/source/reference/changelog.md]: Makes sense!

**Comment by minrk** [comment]
@consideRatio thanks! Updated. It's the end of the day here, so I'll publish 4.0b1 first thing tomorrow.

**Pull request merged**